gearcatalog/frontend/src/components/gear/EditGearForm.tsx

```diff
@@ -25,6 +25,7 @@ match: { params } }: EditGearProps) => {
   const history = useHistory();
   const [gearUpdated, setGearUpdated] = useState(false);
   const [name, setName] = useState(selectedGear.name);
+  const [qty, setQty] = useState(0);
```

Comment:
Why abbreviate the field name?

Reply:
Because I'm lazy and qty works.

Reply:
Laziness is definitely the right thing to base naming on. ;)